package.json

```diff
@@ -1,13 +1,16 @@
 {
   "license": "Apache-2.0",
   "devDependencies": {
-    "@vuepress/client": "2.0.0-rc.0",
-    "@vuepress/plugin-docsearch": "2.0.0-rc.0",
+    "@vuepress/bundler-webpack": "2.0.0-rc.9",
+    "@vuepress/client": "2.0.0-rc.9",
```

Comment:
```suggestion
```

We don't explicitly import `@vuepress/client` and it is already a dependency of `vuepress` so I don't think we should be specifying this.